test/unit/local/steps/linux_producer.js

```diff
@@ -7,166 +7,173 @@ const fs = require('fs')
 const path = require('path')
 const LinuxProducer = require('../../../../core/local/steps/linux_producer')
 
-describe.only('core/local/steps/linux_producer', () => {
-  describe('API of the producer', () => {
-    let syncPath
-    let producer
+os.platform() === 'linux' &&
+  describe.only('core/local/steps/linux_producer', () => {
```

Comment:
What do you think about this code to run tests only on linux platform?

Reply:
I'm not a fan of using `&&` with a lot of lines after: it means that I can't jump in my editor from `{` to `}` to see on what it applies. I know it's a matter of taste (as Python and CoffeeScript avoid blocks by using indentation), but still I don't think it's a good idea here.

And s/describe.only/describe/

Reply:
There is only one instruction here `os.platform() === 'linux' && describe('core/local/steps/linux_producer', linuxProducerTests)`.

But okay, I was not sure anyway. I will update to a more classic if statement:

```
if (os.platform() === 'linux') {
    describe('core/local/steps/linux_producer', linuxProducerTests)
}
```

Reply:
I always forget the `.only`!!!
I am an evil person!

Reply:
Did you think about using the `onPlatform` helper?

Reply:
done